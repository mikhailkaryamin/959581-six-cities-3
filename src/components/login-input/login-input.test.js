import React from "react";
import renderer from "react-test-renderer";
import {
  TypeInputLogin
} from '../../consts';
import LoginInput from './login-input';

const onChange = () => {};

describe(`Login input`, () => {
  test(`Login input email render correctly`, () => {
    const value = `vasya@mail.ru`;
    const tree = renderer
      .create(
          <LoginInput
            type={TypeInputLogin.EMAIL}
            value={value}
            onChange={onChange}
          />
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test(`Login input password render correctly`, () => {
    const value = `ezpz`;
    const tree = renderer
      .create(
          <LoginInput
            type={TypeInputLogin.PASSWORD}
            value={value}
            onChange={onChange}
          />
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});


