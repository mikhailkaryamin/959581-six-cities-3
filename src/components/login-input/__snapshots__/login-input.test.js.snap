// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login input Login input email render correctly 1`] = `
<input
  className="login__input form__input"
  name="email"
  onChange={[MockFunction]}
  placeholder="Email"
  required={true}
  type="email"
  value="vasya@mail.ru"
/>
`;

exports[`Login input Login input password render correctly 1`] = `
<input
  className="login__input form__input"
  name="password"
  onChange={[MockFunction]}
  placeholder="Password"
  required={true}
  type="password"
  value="ezpz"
/>
`;
